src/main/resources/META-INF/plugin.xml

```diff
@@ -11,32 +11,40 @@
 
   <!--    Leave this in here so the IDE gives you feedback on-->
   <!--    What APIs are available-->
-  <idea-version since-build="201.4515.24"/>
+  <idea-version since-build="201.6668.113"/>
 
   <extensions defaultExtensionNs="com.intellij">
-    <applicationService
-      serviceImplementation="zd.zero.waifu.motivator.plugin.settings.WaifuMotivatorPluginState"/>
     <applicationConfigurable
       instance="zd.zero.waifu.motivator.plugin.settings.WaifuMotivatorSettingsPage"/>
+    <applicationService
+      serviceImplementation="zd.zero.waifu.motivator.plugin.settings.WaifuMotivatorPluginState"/>
+    <applicationService serviceImplementation="zd.zero.waifu.motivator.plugin.service.ApplicationService"/>
+    <applicationService serviceImplementation="zd.zero.waifu.motivator.plugin.service.WaifuGatekeeper"/>
     <errorHandler
       id="7111f9b9-f252-483e-9496-72d649347b6d"
       implementation="zd.zero.waifu.motivator.plugin.integrations.ErrorReporter"/>
     <postStartupActivity id="WaifuOfTheDayStartupActivity"
                          implementation="zd.zero.waifu.motivator.plugin.WaifuOfTheDayStartupActivity"/>
-    <applicationService serviceImplementation="zd.zero.waifu.motivator.plugin.service.ApplicationService"/>
-    <applicationService serviceImplementation="zd.zero.waifu.motivator.plugin.service.WaifuGatekeeper"/>
+    <statusBarWidgetFactory implementation="zd.zero.waifu.motivator.plugin.integrations.MoodStatusBarProvider" />
   </extensions>
 
   <actions>
     <group id="zd.zero.waifu.motivator.WaifuMotivatorGroup"
            text="Waifu Motivator" popup="true"
-           icon="WaifuMotivatorIcons.MENU">
+           icon="/icons/wmp.svg">
```

Comment:
Just noticed, weird why this icon is not showing up, while `icon="/icons/heart.svg"` is fine.


<img width="581" alt="Screen Shot 2020-10-23 at 19 45 03" src="https://user-images.githubusercontent.com/21978370/96999748-4991b500-1568-11eb-8122-b912408cccc3.png">

Reply:
This is what I got after tinkering around with it.

![image](https://user-images.githubusercontent.com/15972415/97006454-86789000-1505-11eb-8a87-823a6bf37426.png)

Feel free to adjust as needed :)

Reply:
Found the issue, where it's not about the icon itself, but because of the usage in the new `IconLoader#getIcon` with class parameter. What I'll do is to create a separate ticket and also add an icon for the `Waifu of the Day` so we can have a balanced UI.